assets/javascripts/discourse/components/upcoming-events-list.gjs

```diff
@@ -48,14 +44,21 @@ export default class UpcomingEventsList extends Component {
 
   constructor() {
     super(...arguments);
-
     this.appEvents.on("page:changed", this, this.updateEventsList);
   }
 
   get categoryId() {
     return this.router.currentRoute.attributes?.category?.id;
   }
 
+  get hasEmptyResponse() {
```

Comment:
is there a reason this got moved around?

Reply:
a rebase issue